monai/fl/client/client_algo.py

```diff
@@ -20,43 +20,69 @@ class ClientAlgo(abc.ABC):
     To define a new algo script, subclass this class and implement the
     following abstract methods:
 
-        - #Algo.train()
-        - #Algo.get_weights()
-        - #Algo.evaluate()
+        - self.train()
+        - self.get_weights()
+        - self.evaluate()
 
-    initialize() and finalize() can be optionally be implemented to help with lifecycle management of the object.
+    initialize(), abort(), and finalize() can be optionally be implemented to help with lifecycle management
+    of the class object.
     """
 
     def initialize(self, extra=None):
-        """call to initialize the ClientAlgo class"""
+        """
+        Call to initialize the ClientAlgo class
+
+        Args:
+            extra: optional extra information, e.g. dict of `ExtraItems.CLIENT_NAME` and/or `ExtraItems.APP_ROOT`
```

Comment:
Should we require the type of `extra` in the base class?
For example: `Optional[dict]`.

Thanks.

Reply:
Thanks. I added the typing info.